test/conftest.py

```diff
@@ -899,22 +899,41 @@ def kbase_search_genome_set(
     return save_kbase_search_set(set_api_client, context, genome_refs, ws_id)
 
 
+TYPE_LIST = [
+    ASSEMBLY,
+    f"{DIFFERENTIAL_EXPRESSION_MATRIX}_with_genome",
+    f"{DIFFERENTIAL_EXPRESSION_MATRIX}_no_genome",
+    EXPRESSION,
+    FEATURE_SET,
+    GENOME,
+    READS_ALIGNMENT,
+    READS,
+]
+
+
 @pytest.fixture(scope="session")
 def random_fixture(request: FixtureRequest) -> dict[str, Any]:
     """Pick a random fixture to use for testing."""
-    fixtures = [
-        "assembly_set",
-        "differential_expression_matrix_with_genome_set",
-        "differential_expression_matrix_no_genome_set",
-        "expression_set",
-        "feature_set_set",
-        "genome_set",
-        "reads_alignment_set",
-        "reads_set",
-    ]
+    set_fixtures = [f"{item}_set" for item in TYPE_LIST]
 
     # Randomly select a fixture name
-    selected_fixture = random.choice(fixtures)  # noqa: S311
+    selected_fixture = random.choice(set_fixtures)  # noqa: S311
 
     # Use the request fixture to dynamically get the selected fixture
     return request.getfixturevalue(selected_fixture)
+
+
+@pytest.fixture(scope="session")
+def random_ref(request: FixtureRequest) -> str:
+    """Pick a random reference for something that isn't a set."""
+
+    non_set_refs = [f"{item}_refs" for item in TYPE_LIST]
```

Comment:
uniform naming => easy to generate a list of the names of the refs or the set fixtures